File: app-release-unsigned/sources/com/tsuga/news/MyApplication.java
package com.tsuga.news;

import android.app.Application;
import kotlin.Metadata;
import org.koin.core.context.GlobalContextExtKt;
import org.koin.core.context.KoinContext;

@Metadata(d1 = {"\u0000\u0012\n\u0002\u0018\u0002\n\u0002\u0018\u0002\n\u0002\b\u0002\n\u0002\u0010\u0002\n\u0000\u0018\u00002\u00020\u0001B\u0005¢\u0006\u0002\u0010\u0002J\b\u0010\u0003\u001a\u00020\u0004H\u0016¨\u0006\u0005"}, d2 = {"Lcom/tsuga/news/MyApplication;", "Landroid/app/Application;", "()V", "onCreate", "", "app_release"}, k = 1, mv = {1, 5, 1}, xi = 48)
/* compiled from: MyApplication.kt */
public final class MyApplication extends Application {
    public void onCreate() {
        super.onCreate();
        GlobalContextExtKt.startKoin$default((KoinContext) null, new MyApplication$onCreate$1(this), 1, (Object) null);
    }
}

File: app/src/main/java/com/tsuga/news/MyApplication.kt
package com.tsuga.news

import android.app.Application
import com.tsuga.news.core.di.databaseModule
import com.tsuga.news.core.di.networkModule
import com.tsuga.news.core.di.repositoryModule
import com.tsuga.news.di.useCaseModule
import com.tsuga.news.di.viewModelModule
import org.koin.android.ext.koin.androidContext
import org.koin.android.ext.koin.androidLogger
import org.koin.core.context.startKoin
import org.koin.core.logger.Level

class MyApplication: Application() {
    override fun onCreate() {
        super.onCreate()
        startKoin{
            androidLogger(Level.NONE)
            androidContext(this@MyApplication)
            modules(
                listOf(
                    databaseModule,
                    networkModule,
                    repositoryModule,
                    useCaseModule,
                    viewModelModule
                )
            )
        }
    }
}